### DISCUSSION TITLE
Update license and about section to our template and samples README

### DISCUSSION THREAD

**Discussion by lydiasama** [nimblehq/android-templates#289] [RFCs]
## Why

Most projects in Nimble's repository have the `license` and `about` sections to represent the copyright. So we should add it to our template and samples too.

![Screen Shot 2565-08-22 at 10 40 09 AM](https://user-images.githubusercontent.com/12026942/185834205-316c3f82-d82c-44e0-8ebe-1be104fab31a.png)

Example template that includes this part: 
- https://github.com/nimblehq/ios-templates
- https://github.com/nimblehq/gin-templates
- https://github.com/nimblehq/elixir-templates
- https://github.com/nimblehq/react-templates
- https://github.com/nimblehq/rails-templates

## Who Benefits?

Project owner -- us. 🙌🏻

**Comment by toby-thanathip** [comment]
Resolved in #280
